conf/alluxio-site.properties.template

```diff
@@ -75,6 +75,7 @@ alluxio.user.failed.space.request.limits=3
 alluxio.user.file.buffer.bytes=1MB
 alluxio.user.file.master.client.threads=10
 alluxio.user.file.readtype.default=CACHE_PROMOTE
+alluxio.user.file.cache.incomplete.block=false
```

Comment:
I agree with @calvinjia . A complete (or incomplete) block means something in Alluxio.

instead of "incomplete", maybe "partialread"?